Components/Lists/list.html

```diff
@@ -0,0 +1,242 @@
+                        <li>JS</li>
+                        <li>REACT JS</li>
+                    </ol>
+
+                </div>
+
+                <div class="list ">
+                    <caption>Roman Ordered List</caption>
+                    <div class="divider margin-v-l"></div>
+                    <ol class="roman">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JS</li>
+                        <li>REACT JS</li>
+                    </ol>
+
+                </div>
+
+                <div class="list ">
+                    <caption>Icon List</caption>
+                    <div class="divider margin-v-l"></div>
+                    <ul class="icon">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JS</li>
+                        <li>REACT JS</li>
+                    </ul>
+
+                </div>
+
+
+            </div>
+
+            <!-- ---------------------------------------------------------------- -->
+            <p class="padding-l">To use the any of the above list component, just copy and paste from the below code snippets:</p>
+            <iframe src="https://www.thiscodeworks.com/embed/620625bca4f5770015a88539" style="width: 100%; height: 879px;" frameborder="0"></iframe>
+
+
+            <!-- ------------------------------------------------------------------ -->
+            <div class="divider margin-v-l"></div>
+
+            <h2 class="sub-heading">Stacked lists</h2>
+            <p class="padding-l">Stacked lists are generally seen in the notifications as given below.</p>
+
+            <div class="element-display">
+
+
+                <ul class="stacked-list-container">
+                    <li class="stacked-list">
+                        <ion-icon size="large" name="logo-github"></ion-icon>You have a new notification</li>
+                    <li class="stacked-list">
+                        <ion-icon size="large" name="logo-facebook"></ion-icon>You have a new notification</li>
+                    <li class="stacked-list">
+                        <ion-icon size="large" name="logo-linkedin"></ion-icon>You have a new notification</li>
+                    <li class="stacked-list">
+                        <ion-icon size="large" name="logo-twitter"></ion-icon>
+                        </ion-icon>You have a new notification</li>
+                </ul>
+
+
+            </div>
+
+            <p class="padding-l">To use the above stacked list components, just copy and paste the below code snippets:</p>
+            <iframe src="https://www.thiscodeworks.com/embed/62062617a4f5770015a8853a" style="width: 100%; height: 281px;" frameborder="0"></iframe>
+
+
+            <div class="divider margin-v-l"></div>
+
+            <h2 class="sub-heading">Floated lists</h2>
+            <p class="padding-l">Stacked lists are generally seen in the notifications as given below.</p>
```

Comment:
List description should be for `Floated lists` currently same as `Stacked lists`